### PR TITLE
Interactive list

### PR DESCRIPTION
**Pull request by AdfPizarro**
Include

Localstorage for data persistance
Event listener to change the values

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -39,10 +38,33 @@ function list() {
...
-const listContainer = document.getElementById('todoList');
-listContainer.appendChild(list());
+function toggle(id) {
+  todoList[id].completed = !todoList[id].completed;
+  localStorage.setItem('todoList', JSON.stringify(todoList));
+}
+
+function addListeners() {
+  const navButtons = document.querySelectorAll('.checkbox');
+  navButtons.forEach((el) => el.addEventListener('click', (event) => { // eslint-disable-line no-unused-vars
```
Comment (voscarmv) [inline src/index.js:51]: - [ ] Please remove this and all linter disabling lines. Linter compliance is a basic requirement.

**Review by voscarmv — CHANGES_REQUESTED** [review]
Hi @AdfPizarro , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

- Please read my code comments regarding linter deactivation.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Providing a  live demo link.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AdfPizarro** [comment]
Removed the linters disable

The other disables are necessary 
index.js 1 _  belongs to loadash and will be used only by webpack
`import _ from 'lodash'; // eslint-disable-line no-unused-vars`

index.js line 17 the ++ operator is listed in the es6 documentation https://tc39.es/ecma262/multipage/ecmascript-language-expressions.html#sec-postfix-increment-operator
`  for (let i = 0; i < todoList.length; i++) { // eslint-disable-line no-plusplus`

**Review by carlos-ssh — APPROVED** [review]
Hi @AdfPizarro , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**